packages/webdriver/protocol/appium.json

```diff
@@ -1047,7 +1047,7 @@
       "ref": "http://appium.io/docs/en/commands/interactions/touch/multi-touch-perform/",
       "parameters": [{
         "name": "actions",
-        "type": "object[]",
+        "type": "(array|object)[]",
```

Comment:
I don't think array is correct here. What would `array[]` mean?

Reply:
As explained in the commit message:
> Reason for supporting both is to address inconsistencies between documentation on
Appium (http://appium.io/docs/en/commands/interactions/touch/multi-touch-perform/)
which shows array of objects as JS example, when in tests the expected result is
an array of arrays. This way both options are possible.